src/index.html

```diff
@@ -17,6 +17,86 @@
     />
   </head>
   <body>
+    <header>
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo Logo"
+        />
+      </a>
+      <nav>
+        <ul>
+          <li>
+            <a
+              href="#"
+              class="nav-link is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
```

Comment:
Consider using a more descriptive `href` attribute for the navigation links instead of `#`. This will improve accessibility and provide a meaningful destination for users.